packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -109,7 +109,9 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
             setValidPeer(false)
             setErrorMessage(t('peerList.panels.invalidURL'))
           } else if (response?.data?.peerProfile.hostURL === hostURL) {
-            setValidPeer(false)
+            // TODO for testing! To be removed!
```

Comment:
@penk85 should this be removed?

Reply:
Yes it was for Laurent to peer to self while he tests. I'll set it back.